contracts/core/FreeBets/FreeBetsHolder.sol

```diff
@@ -58,19 +66,17 @@ contract FreeBetsHolder is Initializable, ProxyOwned, ProxyPausable, ProxyReentr
     function fundBatch(address[] calldata _users, address _collateral, uint _amountPerUser) external notPaused nonReentrant {
         require(supportedCollateral[_collateral], "Unsupported collateral");
         IERC20(_collateral).safeTransferFrom(msg.sender, address(this), _amountPerUser * _users.length);
-        for (uint256 index = 0; index < _users.length; index++) {
+        for (uint256 index; index < _users.length; ++index) {
```

Comment:
no need to explicitely use `uint256 `, just use `uint`.
`index` is often a reserved word, so I think its not the best name for an iterator or any variable.